ui/client/components/toolbars/process/buttons/ArchiveButton.tsx

```diff
@@ -41,7 +41,7 @@ function ArchiveButton(props: StateProps) {
 const mapState = (state: RootState) => {
   return {
     processId: getProcessId(state),
-    isRunning: isRunning(state),
+    canArchive: !isSubprocess(state) && isArchivePossible(state) || isSubprocess(state),
```

Comment:
let's add parentheses here... or can we just write:
 isArchivePossible(state) || isSubprocess(state)??

Reply:
We can't because subprocess can't get process state..

Reply:
so at least  isSubprocess(state) || isArchivePossible(state)
- it's equivalent, isn't it?